WSL/wsl-config.md

```diff
@@ -2,28 +2,24 @@
 title: Manage Linux Distributions
 description: Reference listing and configuring multiple Linux distributions running on the Windows Subsystem for Linux.
 keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, wsl.conf, wslconfig
-ms.date: 02/7/2018
+ms.date: 05/19/2020
 ms.topic: article
-ms.assetid: 7ca59bd7-d9d3-4f6d-8b92-b8faa9bcf250
-ms.custom: seodec18
-ms.localizationpriority: high
 ---
 
-# Manage and configure Windows Subsystem for Linux
-
-> Applies to Windows 10 Fall Creators Update and later.  See our updated [installation guide](./install_guide.md) to try new management features and start running multiple Linux distributions from the Microsoft store.
+# WSL commands and launch configurations
 
 ## Ways to run WSL
 
-There are many ways to run Linux with the Windows Subsystem for Linux.
+There are several ways to run a Linux distribution with WSL once it's [installed](install-win10.md).
 
-1. `[distro]`, for example `ubuntu`
-1. `wsl.exe` or `bash.exe`
-1. `wsl [command]` or `bash -c [command]`
+1. Open your Linux distribution by visiting the Windows Start menu and typing the name of your installed distributions. For example: "Ubuntu".
+2. From Windows Command Prompt or PowerShell, enter the name of your installed distribution. For example: `ubuntu`
+3. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter: `wsl.exe` or `bash.exe`
```

Comment:
We're trying to move users to using `wsl.exe`, `bash.exe` is old and doesn't have as many features.

```suggestion
3. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter: `wsl.exe`.
```